finality-aleph/src/abft/crypto.rs

```diff
@@ -105,8 +105,8 @@ impl current_aleph_bft::MultiKeychain for Keychain {
         index: current_aleph_bft::NodeIndex,
     ) -> Self::PartialMultisignature {
         current_aleph_bft::PartialMultisignature::add_signature(
-            SignatureSet(legacy_aleph_bft::SignatureSet::with_size(
-                legacy_aleph_bft::Keychain::node_count(self),
+            SignatureSet(aleph_bft_crypto::SignatureSet::with_size(
```

Comment:
I noticed that in both `current` and `legacy` implementations of `MultiKeychain`, legacy signature sets were used (which simply used older `aleph-bft-crypto`). I understood that we want the same signing interface regardless of legacy / current. As there were changes in `aleph-bft-crypto`, I decided to use the newest `aleph-bft-crypto` for that (which currently matches `current_aleph_bft::SignatureSet`).

Reply:
This depends, what kind of changes where there in aleph-bft-crypto? ANy changes to the signatureset encoding/decoding?

Reply:
I see no such changes

Reply:
It doesn't compile with anything being `legacy`. This is due to line 39 in `abft/mod` in which we declare `SignatureSet` parametrized by `aleph_bft_crypto::SignatureSet`. I mentioned that previously, `legacy_aleph_bft` prefix was used in both `Current` and `Legacy` multikeychains, which was equivalent to `aleph_bft_crypto` prefix, as legacy abft used `aleph-bft-crypto` from workspace. Now, in this pr we update `aleph-bft-crypto` to the newest version, which is equivalent to version used by `current_aleph_bft`. So we need to use it, or change the `SignatureSet` but it doesn't seem good.

Reply:
all good here